### PR TITLE
Features/contact page

### PR DESCRIPTION
**Pull request by Dolybim**

### CODE REVIEW

File: src/Pages/Contact.tsx [hunk 1]
```diff
@@ -26,175 +32,186 @@ const Contact: React.FC = () => {
 // const [email, setEmail] = useState('');
 // const [message, setMessage] = useState('');
```
Comment (OluyideO) [inline src/Pages/Contact.tsx:33]: Let us remove these comments if they are not useful

**Review by OluyideO — APPROVED** [review]

File: src/Pages/Contact.tsx [hunk 2]
```diff
@@ -22,128 +32,186 @@ const Contact: React.FC = () => {
...
+              message: '',
+            }}
+
+            onSubmit={(values, { setSubmitting }) => {
+              setTimeout(() => {
+                console.log("Loggin in", values)
+              }, 500)
+            }}
+
+            // onSubmit={(values, actions) => handleSubmit(values.email, values.message)
```
Comment (OluyideO) [inline src/Pages/Contact.tsx:102]: let us remove unncessary comments

**Review by OluyideO — CHANGES_REQUESTED** [review]

**Comment by OluyideO** [comment]
There is a conflict in  this branch please resolve conflicts

File: src/Pages/Contact.tsx (hunk 1 above)
Comment (Dolybim) [reply src/Pages/Contact.tsx:33]: alright, done

File: src/ourTeam.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@media screen and (max-width: 640px) {
+    .header-wrapper {
+        /* height: 100vh; */
```
[the remark this answers is not in the document]
Comment (Dolybim) [reply src/ourTeam.css:3]: done

**Review by OluyideO — APPROVED** [review]

**Pull request merged**